Grizzlies-SpyDuh/Repository/UserRepository.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Grizzlies_SpyDuh.Utils;
+using Microsoft.Data.SqlClient;
+using Microsoft.Extensions.Hosting;
+using System.Collections.Generic;
+
+namespace Grizzlies_SpyDuh.Repository
+{
+    public class UserRepository //: BaseRepository, IUserRepository
```

Comment:
Base repository can be uncommented. Currently there are errors because you are using Connection on line 13, which doesn't exist if there's no base repository.